R/TreeViz-methods.R

```diff
@@ -30,35 +30,48 @@ setGeneric("aggregateTree", signature = "x",
 #' @param start,end indices to filter nodes
```

Comment:
I think all changes in this file can be safely ignored. since boxplots use a different method